LayoutTests/accessibility/element-reflection.html

```diff
@@ -0,0 +1,41 @@
+<!DOCTYPE html>
+<script src="../resources/js-test-pre.js"></script>
```

Comment:
we no longer need the pre and post test js files

you can do

```
<!DOCTYPE HTML PUBLIC "-//IETF//DTD HTML//EN">
<html>
<head>
<script src="../resources/accessibility-helper.js"></script>
<script src="../resources/js-test.js"></script>
</head>
<body>
```

Reply:
Done.